### PR TITLE
Update nonlocal.rst

### PR DESCRIPTION
**Pull request by pcphp**
a small typo mistyped

### CODE REVIEW

File: manual/sphinx/user_docs/nonlocal.rst [hunk 1]
```diff
@@ -18,7 +18,7 @@ The Spitzer-Harm heat flux `q_{SH}` is calculated using
 where `n_e` is the electron density in `m^{-3}`, `T_e` is the electron temperature in eV, `kappa_0 = 13.58`,
```
Comment (dschwoerer) [inline manual/sphinx/user_docs/nonlocal.rst:18]: Here is another issue:
```suggestion
where `n_e` is the electron density in `m^{-3}`, `T_e` is the electron temperature in eV, `\kappa_0 = 13.58`,
```

File: manual/sphinx/user_docs/nonlocal.rst [hunk 2]
```diff
@@ -18,7 +18,7 @@ The Spitzer-Harm heat flux `q_{SH}` is calculated using
...
 `Z` is the average ion charge. The resulting expression is in units of `eV/m^2/s`. 
 
-The thermal collision time `tau_{ei,T} = \lambda_{ei,T} / v_{T}` is calculated using the thermal mean free path
+The thermal collision time \tau_{ei,T} = \lambda_{ei,T} / v_{T}` is calculated using the thermal mean free path
```
Comment (dschwoerer) [inline manual/sphinx/user_docs/nonlocal.rst:21]:
```suggestion
The thermal collision time `\tau_{ei,T} = \lambda_{ei,T} / v_{T}` is calculated using the thermal mean free path
```
The ` is additionally need for a math environment

**Review by dschwoerer — CHANGES_REQUESTED** [review]
Thanks a lot for noting and the PR :+1:

**Comment by bendudson** [comment]
Thanks @pcphp and @dschwoerer !

**Review by bendudson — APPROVED** [review]

**Pull request merged**